README.md

```diff
@@ -123,15 +123,15 @@ Google Cloud BigQuery (Alpha)
 
 Here is a code snippet showing a simple usage example from within Compute/App Engine. Note that you
 must [supply credentials](#authentication) and a project ID if running this snippet elsewhere.
+Complete source code can be found at
```

Comment:
I think the links need to also be modified in the modules' READMEs, like [here](https://github.com/mziccard/gcloud-java/blob/refactor-examples/gcloud-java-bigquery/README.md#example-application).